os/se/ameba/rtl_security_api_wrapper_tz.c

```diff
@@ -79,6 +80,13 @@ static void ns_setstatusbits(u32 NewState)
 	device_mutex_unlock(RT_DEV_LOCK_FLASH);
 }
 
+static int ns_printf(const char *input)
+{
+	int ret;
+	ret = sedbg("%s \n", input);
```

Comment:
I think `return sedbg("%s \n", input);` is enough.

Reply:
As I left before, when se debug config is disabled, sedbg will be nothing.
```
#ifdef CONFIG_DEBUG_SECURE_ELEMENT_ERROR
#define sedbg(format, ...)     dbg(format, ##__VA_ARGS__)
#define selldbg(format, ...)   lldbg(format, ##__VA_ARGS__)
#else
#define sedbg(...)
#define selldbg(...)
#endif
```
Then `return sedbg(...);` will be `return ;`. I think it also prints build error.

Reply:
I have change it to return void.